redaxo/src/addons/be_style/plugins/redaxo/scss/_navs.scss

```
@@ -227,53 +227,47 @@
 
 .rex-page-nav {
     .nav-tabs {
-        border-bottom-color: $color-a-light;
+        border-bottom-color: darken($color-a-light, 5%);
         .rex-main-frame & {
             border-bottom-color: #fff;
         }
 
         > li {
-            .rex-main-frame & {
-                margin-bottom: 0;
-            }
-
             &.active {
                 > a:not(.btn),
                 > a:not(.btn):hover {
+                    padding-bottom: 10px;
                     background-color: #fff;
-                    border-top-color: $color-b;
-                    border-top-width: 2px;
+                    border-color: darken($color-a-light, 5%);
+                    border-bottom-color: transparent;
 
                     .rex-main-frame & {
-                        border-left-width: 0;
-                        border-right-width: 0;
-                        border-top-color: $color-b;
+                        background-color: #fff;
+                        border-color: #fff;
```

```suggestion
                        background-color: $nav-tabs-active-link-hover-bg;
                        border-color: $nav-tabs-active-link-hover-bg;
```
Ist zwar  nicht ganz richtig, aber BS3 hat keine andere Var.